src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/DatabaseMaintainer.java

```diff
@@ -0,0 +1,69 @@
+package edu.kit.ipd.crowdcontrol.objectservice.database;
+
+import org.jooq.DSLContext;
+
+import java.io.Closeable;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import static edu.kit.ipd.crowdcontrol.objectservice.database.model.Tables.*;
+
+/**
+ * this class periodically performs cleanup-tasks.
+ * @author LeanderK
+ * @version 1.0
+ */
+public class DatabaseMaintainer {
+    private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    private final DSLContext create;
+    private final int cleanUpInterval;
+    private ScheduledFuture<?> schedule = null;
+
+    /**
+     * creates a new Instance of DatabaseMaintainer.
+     * @param create the context used to communicate with the database
+     * @param cleanUpInterval the intervall the task should be run, in hours
+     */
+    public DatabaseMaintainer(DSLContext create, int cleanUpInterval) {
+        this.create = create;
+        this.cleanUpInterval = cleanUpInterval;
+    }
+
+    /**
+     * starts the DatabaseMaintainer
+     */
+    public synchronized void start() {
+        if (schedule != null) {
+            schedule = scheduler.schedule(this::doCleanUp, cleanUpInterval, TimeUnit.HOURS);
```

Comment:
this should be: scheduler.scheduleAtFixedRate(this::doCleanUp, 0, cleanUpInterval, TimeUnit.HOURS);